src/einsteinpy/utils/christoffel.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from sympy import Matrix, diff, eye, sin, symbols, zeros
+
+
+def christoffels(list2d, syms):
```

Comment:
Can we rename the function so that the utility becomes more clear to the user?

Reply:
Any noob user won't come here.
Only those who know christoffel symbols will use this, and therefore, i think the name is apt.

Reply:
> Only those who know christoffel symbols will use this

To make their lives more easy!